javascript/javascript2/week1/shortestWord.js

```diff
@@ -0,0 +1,57 @@
+const danishWords = ['bil', 'plante', 'kaffe', 'bog', 'ø', 'planetarium'];
+const danishString = 'Jeg har en blå bil';
+
+
+
+const finalObj={};
+function shortesWord(inputWords){
+    let wordCheker= inputWords[0];
+    //inputWords = danishWords;
+    
+    if ( inputWords instanceof Array){
```

Comment:
May be you don't need to check of the instanceOf as you can have 2 diffrent function, in homework it was 2 diffrent statements.